helm/Chart.yaml

```diff
@@ -24,4 +24,4 @@ version: 0.1.0
 dependencies:
   - name: config-bootstrapper
     repository: "https://storage.googleapis.com/hypertrace-helm-charts"
-    version: 0.2.20
+    version: 0.2.21
```

Comment:
I know this is required only for attribute service, should we also update at Entity Service too, just to be on the same version of config-bootstraper on both the hook - https://github.com/hypertrace/entity-service/blob/main/helm/Chart.yaml#L27

Reply:
CC @aaron-steinfeld

Reply:
It makes sense. We know there's no reason to and it won't matter since we saw the config bootstrapper change - but aligning them will help the next person who doesn't have the context.

Reply:
This repo is failing snyk though, can we fix that - either in this PR or a separate one. Easiest fix will be to upgrade grpc and grpc-utils, we fixed this vulnerability in the utils (and we want to keep the grpc version aligned with the utils).

Reply:
let's update as part of this PR.

Reply:
updated